src/control/server/config_bdev.go

```diff
@@ -54,84 +57,157 @@ const (
 `
 	gbyte   = 1000000000
 	blkSize = 4096
+
```

Comment:
This patch still needs to be rebased on #405 right? these seem like additions from that patch, might have some conflicts

Reply:
done